blazor/stock-chart/tool-tip.md

```diff
@@ -157,4 +157,71 @@ The [Fill](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.StockC
 
 ```
 
-![Blazor Stock Chart with Custom Tooltip](images/common/blazor-stock-chart-custom-tooltip.png)
\ No newline at end of file
+![Blazor Stock Chart with Custom Tooltip](images/common/blazor-stock-chart-custom-tooltip.png)
+
+## Moving Tooltip
```

Comment:
@Bharathipriyansf3872 Modify like ej2 https://ej2.syncfusion.com/documentation/stock-chart/tool-tip#position-the-tooltip, Also add gif image instead of normal image.